timeseries/src/autogluon/timeseries/models/abstract/abstract_timeseries_model.py

```diff
@@ -32,9 +41,88 @@
 from .model_trial import model_trial, skip_hpo
 
 logger = logging.getLogger(__name__)
+dup_filter = DuplicateFilter()
+logger.addFilter(dup_filter)
+
 
+# TODO: refactor and move to util. We do not need to use "params_aux" in time series
```

Comment:
Not sure if `params_aux` would be the best tool for that, but it would be nice to keep the mechanism for setting `max_time_limit` for individual models

Reply:
Noted. Now that `params_aux` is not really needed, we can also think of other things lurking as hyperparameters and constructor arguments and pull them into an immutable dataclass..